src/jit/lsraxarch.cpp

```diff
@@ -2261,7 +2261,7 @@ void LinearScan::TreeNodeInfoInitSIMD(GenTreeSIMD* simdTree)
                 {
                     bool needFloatTemp;
                     if (varTypeIsSmallInt(simdTree->gtSIMDBaseType) &&
-                        (compiler->getSIMDInstructionSet() == InstructionSet_AVX))
```

Comment:
Isn't this change part of a different PR?

Reply:
Please see https://github.com/dotnet/coreclr/pull/13879